config/jobs/kubernetes/sig-security/govulncheck-presubmit.yaml

```diff
@@ -0,0 +1,37 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: check-govulncheck-results
+    cluster: eks-prow-build-cluster
+    decorate: true
+    decoration_config:
+      timeout: 5m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+      - org: kubernetes
+        repo: sig-security
+        base_ref: main
+        workdir: true
+    always_run: false
+    optional: true
+    run_if_changed: '^(go.mod|go.sum|vendor)'
+    spec:
+      containers:
+      - image: golang
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
+          cd ../kubernetes
```

Comment:
I believe you can get rid of this line if you set `k/k` in `extra_refs`. i.e.

```
      extra_refs:
      - org: kubernetes
        repo: kubernetes
        base_ref: main
        workdir: true
      - org: kubernetes
        repo: sig-security
        base_ref: main

```

Reply:
But this is a presubmit...by adding `kubernetes` in `extra_refs` will it not overwrite the PR specific clone of `k/k`?

Reply:
gah, yes. You're right :)